src/test/java/uk/ac/ebi/ena/webin/cli/WebinCliBuilder.java

```diff
@@ -77,6 +95,13 @@ public WebinCli build(Path inputDir, ManifestBuilder manifestBuilder) {
     return new WebinCli(WebinCliTestUtils.getTestWebinUsername(),getAuthToken(), cmd);
   }
 
+  public WebinCli build(File inputDir, ManifestBuilder manifestBuilder, boolean ignoreErrors) {
```

Comment:
- ```WebinCliParameters``` should only be created in ```WebinCli.initParameters``` (except in unit tests).
- This ```build``` functionality is called only from tests.
-  We should not add test only functionality into WebinCliBuilder.
- Please remove this ```build``` from WebinCliBuilder.
- Why not just call ```WebinCli(WebinCliParameters parameters)``` from unit tests instead of using ```WebinCliBuilder```.